crates/aptos-telemetry-service/src/metrics.rs

```diff
@@ -142,10 +143,15 @@ impl PrometheusExporter {
         ];
 
         let start_timer = Instant::now();
-
+        let timestamp = Utc::now().timestamp() as usize;
```

Comment:
I would suggest confirming if the timestamp is indeed in Utc, otherwise we might corrupt data unnecessarily.

Reply:
Good catch. The documentation (https://docs.victoriametrics.com/#timestamp-formats) says unix timestamps. I changes from UTC to unix timestamps.